src/settings/components/SectionPageItem.js

```diff
@@ -5,7 +5,7 @@ import {
   NavListItem,
 } from '@folio/stripes/components';
 
-const SectionPageItem = ({ setting, path }) => {
+const SectionPageItem = ({ setting, path, hasInterface }) => {
```

Comment:
Let's avoid re-implementing `hasInterface` in `SettingsPage` and passing it as a prop. We can just use the original:
```
const stripes = useStripes();
// ...
if (stripes.hasInterface(setting?.dependsOnNoneInterface)) { ... }
```